### DISCUSSION TITLE
Deliver: How to Release App Already Reviewed

### DISCUSSION THREAD

**Discussion by mieliespoor** [fastlane/fastlane#18480] [Q&A] [answered]
We submit our apps for review using deliver and then once the app is "Pending Developer Release", we make a decision when to release using phased release.

The submit for review using deliver looks like this:

```
    deliver(
      api_key: api_key,
      build_number: "#{appVersion}",
      app_identifier: "#{bundleid}",
      app_version: "#{version}",
      team_id: "#{team_id}",
      team_name: "#{team_name}",
      submit_for_review: true,
      automatic_release: false,
      force: true, # Skip HTMl report verification
      skip_metadata: true,
      skip_screenshots: true,
      skip_binary_upload: true,
      phased_release: true,
      precheck_include_in_app_purchases: false,
      release_notes: release_notes,
      submission_information: {
        add_id_info_uses_idfa: false
      }
    )
```
*note. We use an already uploaded testflight build.

It's not clear what we should then do to release the app. I can't set `automatic_release=true` because it has already been reviewed. I also don't want to submit it again.

**Comment by kambala-decapitator** [comment, marked as answer]
see here: https://github.com/fastlane/fastlane/issues/17140#issuecomment-686493328
